packages/react/src/components/DatepickerInput/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../Icon';
+
+const DatepickerInput = React.forwardRef(
+  ({ onClick, placeholder, value, isInvalid }, ref) => (
+    <div
+      role='button'
+      tabIndex={0}
+      ref={ref}
+      onClick={onClick}
+      onKeyPress={({ which }) => (which === 13 ? onClick() : undefined)}
+      className='f-FormEl f-FormEl--withIcon f-Datepicker'
+    >
+      {value || placeholder}
+      <Icon
+        icon={isInvalid ? Icon.ICONS.IconAlertOctagon : Icon.ICONS.IconCalendar}
+        className='f-FormEl-icon'
+      />
+    </div>
+  ),
+);
+
+DatepickerInput.displayName = 'DatepickerInput';
+
+DatepickerInput.propTypes = {
+  onClick: PropTypes.func,
+  placeholder: PropTypes.string,
+  value: PropTypes.string,
+  isInvalid: PropTypes.bool,
+};
+
+DatepickerInput.defaultProps = {
+  onClick: PropTypes.func,
+  placeholder: 'Select a date',
+  value: null,
+  isInvalid: false,
```

Comment:
Please remove these too 🙏

Reply:
We still have `react/require-default-props` as an eslint rule so I'll keep the default props for now

Reply:
Ah right. I removed that rule for V2. I'll remove the defaults in the migration then